ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/macro/MacroRegistryImpl.java

```diff
@@ -39,9 +38,7 @@ public MacroRegistryImpl() {
     public void register(Set<Macro> macros) {
         for (Macro macro : macros) {
             final String name = macro.getName();
-            if (this.macros.containsKey(name)) {
-                Log.warn(MacroRegistryImpl.class, "Command macro '" + name + "' is already registered.");
```

Comment:
@vitaliy-guliy it's really nice that you're worried about cleanliness of the browser's DevConsole and you can propose your suggestions for discussion by team in another PR